ui/src/color_picker.tsx

```diff
@@ -111,18 +120,19 @@ export const
       [selectedColorId, setSelectedColorId] = React.useState<S | null>(defaultValue),
       onSwatchChange = (_e: React.FormEvent<HTMLElement>, _id?: S, color = defaultValue) => {
         wave.args[name] = color
+        model.value = color || undefined
```

Comment:
Why `undefined`? Would updating the color value from the initial `None` to a `None` work on the Python side?

Reply:
The `color` prop of `onSwatchChange` will always have value since we does not support unselection (swatch picker will always have some value `selectedId={selectedColorId || choices[0]}`). So using `undefined` is just to match types.
 
https://github.com/h2oai/wave/assets/23740173/f19e1d62-d620-4842-bc26-19e12ad9e513